test/integration/model/result/messages/LisOutcomeRequestParserTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoLtiConsumer\test\integration\model\result\messages;
+
+use oat\generis\test\MockObject;
+use oat\generis\test\TestCase;
+use oat\taoLtiConsumer\model\result\messages\LisOutcomeRequestParser;
+use oat\taoLtiConsumer\model\result\operations\OperationsCollection;
+use oat\taoLtiConsumer\model\result\operations\replace\OperationRequest;
+use oat\taoLtiConsumer\model\result\operations\replace\OperationRequestParser;
+
+class LisOutcomeRequestParserTest extends TestCase
+{
+    /** @var OperationsCollection|MockObject */
+    private $operationsCollectionMock;
+
+    public function testParseLti1p3OutcomeBasic()
+    {
+        $payload = $this->getPayload();
+        $subject = new LisOutcomeRequestParser();
+        $this->operationsCollectionMock = $this->createMock(OperationsCollection::class);
+        $this->operationsCollectionMock
+            ->method('getOperationRequestParser')
+            ->willReturn(new OperationRequestParser());
+        $subject->setServiceLocator(
+            $this->getServiceLocatorMock(
+                [
+                    OperationsCollection::class => $this->operationsCollectionMock,
+                ]
+            )
+        );
+        $xml = $subject->parse($payload);
+        $this->assertInstanceOf(OperationRequest::class, $xml->getOperation());
+        $this->assertSame('lisResultSourcedId', $xml->getOperation()->getSourcedId());
+        $this->assertSame('replaceResultRequest', $xml->getOperationName());
+        $this->assertSame('messageIdentifier', $xml->getMessageIdentifier());
+    }
+
+    private function getPayload(): string
+    {
+        return '<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
i am afraid that this might not be proper, investigating now

Reply:
Yes. It needs to be changed

Reply:
The namespace is correct with this one: https://github.com/oat-sa/extension-tao-lti/pull/269/files#diff-d00bad1f349618d6aa1dcca42983e600R38

Reply:
let's try to use the : https://github.com/oat-sa/extension-tao-lti/pull/269 as a single point of truth for all the other extensions